packages/now-next/src/utils.ts

```diff
@@ -455,9 +457,13 @@ export const ExperimentalTraceVersion = `9.0.4-canary.1`;
 
 export type PseudoLayer = {
   [fileName: string]: {
-    crc32: number;
-    compBuffer: Buffer;
-    uncompressedSize: number;
+    crc32?: number;
+    compBuffer?: Buffer;
+    uncompressedSize?: number;
+
+    file?: FileFsRef;
+    isSymlink?: boolean;
+    symlinkTarget?: string;
```

Comment:
I think you you can be more specific here:

```ts
export type PseudoLayer = {
  [fileName: string]: PseudoFile | PseudoSymbolicLink
}

export type PseudoFile = {
  isSymlink: false;
  crc32: number;
  compBuffer: Buffer;
  uncompressedSize: number;
}

export type PseudoSymbolicLink = {
  isSymlink: true;
  file: FileFsRef;
  symlinkTarget?: string;
}
```

Reply:
That looks better 👍